webaccess/src/webaccess.cpp

```diff
@@ -1549,7 +1562,7 @@ QString WebAccess::getCueListHTML(VCCueList *cue)
         {
             QString stepID = QString::number(cue->id()) + "_" + QString::number(i);
             str += "<tr id=\"" + stepID + "\" "
-                   "onclick=\"enableCue(" + QString::number(cue->id()) + ", " + QString::number(i) + ");\">\n";
```

Comment:
Why? setCueIndex doesn't send the feedback back to the C++ code

Reply:
In the C++ version, the cuelist does not play when you click the step.
But in the web version, the cuelist played when you clicked the step.
So I changed the the enableCue function to setCueIndex.

What do you think?
I am open to discuss this.